build/configs/artik053/st_things/defconfig

```diff
@@ -1163,6 +1163,7 @@ CONFIG_EXAMPLES_ST_THINGS_LIGHT=y
 # CONFIG_EXAMPLES_ST_THINGS_MSG_HANDLING is not set
 CONFIG_RESET_BUTTON=y
 CONFIG_RESET_BUTTON_PIN_NUMBER=44
+CONFIG_HANDLE_GPIO_EVENT_NUMBER=42
```

Comment:
please add kconfig in sample_light

Reply:
Could you please explain more detail? 
You mean CONFIG_HANDLE_GPIO_EVENT_NUMBER should be able to set any PIN Number by user?

Reply:
I saw in description of artik053 board. The default value for  switch SW702 is gpio no 42.